book/scheduling-and-threading.md

```diff
@@ -573,19 +571,25 @@ class Browser:
             draw_timer.start()
             self.draw()
             self.time_in_draw += draw_timer.stop()
+            self.time_in_raster_and_draw += timer.stop()
         self.needs_tab_raster = False
         self.needs_chrome_raster = False
         self.needs_draw = False
-        if timer:
-            self.time_in_raster_and_draw += timer.stop()
 
     def handle_quit(self):
-        print("Time in raster and draw: {:>.6f}s".format(
-            self.time_in_raster_and_draw))
-        print("Time in draw: {:>.6f}s".format(
-            self.time_in_draw))
-
-        self.tabs[self.active_tab].handle_quit()
+        print("""Time in raster-and-draw: {:>.6f}s
+    ({:>.6f}ms per raster-and-draw run on average;
+    {} total raster-and-draw updates)""".format(
```

Comment:
Not sure what you mean, do you mean the debug output?

Reply:
Instead of showing raster and raster+draw time, it would be nice if we showed raster and draw time. (This means subtracting the raster time from the raster+draw time.)